internal/application/tenant/service_test.go

```
@@ -0,0 +1,176 @@
+	}
+	defer db.Close()
+
+	mock.ExpectBegin()
+	mock.ExpectRollback()
+
+	service := NewUpdateTenantService(tenantRepoStub{
+		findErr: ErrTenantNotFound,
+	}, dbtxrunner.New(db, nil))
+
+	name := "Tenant A"
+	err = nil
```

golangci-lint (2.11.4)

[error] ineffectual assignment to err

(ineffassign)